src/main/resources/config.yml

```diff
@@ -1,26 +1 @@
-# 
```

Comment:
I don't agree with removing this. Although it is created at run time it is very useful to reference and look at. It can be inferred from Settings but having a file available is useful. It does mean that it needs to be updated when the Settings are updated but that's okay. So, please keep this file.